### PR TITLE
Alok 38

### PR DESCRIPTION
**Pull request by alok-38**
# 🛠️ Fixes issue # 49
Improved the aesthetic and the readability of the README.md file.
# 📇 Changes proposed 
Some aesthetic changes were proposed. Updated the same post approval.
# ✅ Check List (Mark all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] This PR contains changes in the documentation
- [x] This PR contains changes in the Code of the project
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers
Please suggest/advice if more enhancements are needed.

## 📷 Screenshots
Screenshot from 2023-01-02 20-30-03

### CODE REVIEW

**Comment by hasnainmakada-99** [comment]
> # 🛠️ Fixes issue #
> Improved the aesthetic and the readability of the README.md file.
> # 📇 Changes proposed 
> Some aesthetic changes were proposed. Updated the same post approval.
> # ✅ Check List (Mark all the applicable boxes)
> 
> 
> 
> - [x] My code follows the code style of this project.
> - [x] This PR does not contain plagiarized content.
> - [x] This PR contains changes in the documentation
> - [x] This PR contains changes in the Code of the project
> - [x] The title of my pull request is a short description of the requested changes.
> 
> ## 📄 Note to reviewers
> Please suggest/advice if more enhancements are needed.
> 
> 
> 
> ## 📷 Screenshots
> Screenshot from 2023-01-02 20-30-03
> 

Please make the pr title more clear on what changes does this pr make and please dont edit the allcontributors file, I only have the rights that's why and please revoke the changes made to the all-contributors.src file

**Comment by hasnainmakada-99** [comment]
And which issue does this pr refer to, please mention it

File: .all-contributorsrc [hunk 1]
```diff
@@ -104,6 +104,15 @@
       "contributions": [
         "design"
       ]
+    },
```
Comment (hasnainmakada-99) [inline .all-contributorsrc:107]: Not Required

**Comment by alok-38** [comment]
I have updated the issue number. I think I have missed out on this on a couple of occasions. I apologize for the same. It won't happen again.

**Comment by hasnainmakada-99** [comment]
> I have updated the issue number. I think I have missed out on this on a couple of occasions. I apologize for the same. It won't happen again.

No Problem @alok-38 , Take your time to work on this

**Comment by alok-38** [comment]
I touched the ```.all-contributorsrc``` by mistake. How do I revoke it?

**Pull request closed without merging**